iCubGenova09/hardware/electronics/face-eb22-j0-eln.xml

```diff
@@ -22,7 +22,7 @@
                 <param name="maxTimeOfRXactivity">      400                 </param>
                 <param name="maxTimeOfDOactivity">      300                 </param>   
                 <param name="maxTimeOfTXactivity">      300                 </param>                
-                <param name="TXrateOfRegularROPs">      3                   </param> 
+                <param name="TXrateOfRegularROPs">      2                   </param> 
```

Comment:
Do you have any reference for this change?

Reply:
From [here](https://github.com/robotology/robots-configuration/blob/f4be4b92aba5717dedd13677fe43f7c6b3a3ab5e/iCubTemplates/iCubTemplateV6_0/hardware/electronics/body_part--ebX-jA_B-eln.xml#L62-L73)
```

                <!-- The period in multiples of RUNNINGMODE::period with which the ETH boards transmits the regular ROPs. Its range is [1, 20] and if out of bound it is clipped to limits. 
                     The recommended value is 5 (3 for boards which support skin). IMPORTANT: reply ROPs and diagnostics ROPs are transmitted ASAP with 1 ms granularity.
                                  
                     This number is related to many other system parameters: (1) the ETH_BOARD::ETH_BOARD_SETTINGS::regularsTXrate of other boards; (2) the PC104::PC104RXrate decoding 
                     rate inside PC104; (3) rate of the wrappers which typically is 10 ms. 
                     A value of PC104::PC104RXrate = 1 for every board is possible, but it adds extra useless effort in the CPU of PC104 to decode many UDP packets. 
                     The effort is useless because the information carried by regular ROPs is read by wrappers typically at 10 ms rate.
                     hence, any value lower than the rate of wrappers is enough. The only exception is for presence of skin, where it is required a lower value so that the ETH 
                     board can consume the CAN packets sent by the CAN boards without losing info (if it happens, diagnostics messages are sent which complain about loss of skin data).  
                  -->   
```